acct-user/graylog/graylog-0.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="Graylog program user"
+ACCT_USER_ID=478
+ACCT_USER_HOME=/var/lib/graylog
```

Comment:
If possible, please leave `ACCT_USER_HOME` at the default and create the directory in the ebuild that uses it. This will make it easier for other people to share this user account in the future (without having to fight over the home directory).

Reply:
I cannot imagine any other program using the graylog user :)

Reply:
Lack of imagination =P

Before I added `app-antivirus/clamav-unofficial-sigs` to the tree, I'm sure the maintainer of `app-antivirus/clamav` didn't think there was any reason for someone else to ever use the "clamav" user. But now there is, and I do.

I like to make up numbers, so: 75% of the motivation for GLEP81 was that these user accounts could be shared if some day there was a need to.

Reply:
Ok, I'll move it to the ebuild then :)

Reply:
Thank you. I've tried really hard to get these guidelines into the devmanual (to avoid making people re-do ebuilds), but it's moving at a glacial pace even after two rounds of review on the mailing list.

Reply:
Thanks for taking your time on this, no problem to update the ebuild.